src/studio-header/HeaderBody.test.tsx

```diff
@@ -35,7 +35,7 @@ const defaultProps = {
 
 const RootWrapper = (props) => (
   <MemoryRouter>
-    <IntlProvider locale="en" messages={messages}>
+    <IntlProvider locale="en" messages={{}}>
```

Comment:
Could you elaborate on this change? It's not clear to me why the `messages` imported from `./messages` are no longer being passed in here.

Reply:
Because TypeScript was reporting a type error.

The `messages` imported from `messages` are in a different format from the `messages` that `IntlProvider` accepts (I think basically the latter are compiled messages; the former are not compiled). In any case, our test cases generally [don't pass any `messages` to `IntlProvider`](https://github.com/search?q=repo%3Aopenedx%2Ffrontend-app-authoring+IntlProvider+messages&type=code) because it wouldn't have any effect - the message string is already in the source file (via `defaultMessages`) and the tests aren't localized so there is no need for passing any message strings in, unless you wanted to test localization into another language.